be/src/storage/tablet_updates.cpp

```diff
@@ -1122,21 +1178,23 @@ Status TabletUpdates::primary_index_dump(PrimaryKeyDump* dump, PrimaryIndexMulti
     return Status::OK();
 }
 
-void TabletUpdates::_apply_rowset_commit(const EditVersionInfo& version_info) {
+Status TabletUpdates::_apply_rowset_commit(const EditVersionInfo& version_info) {
```

Comment:
How about refactor these three apply function (`_apply_column_partial_update_commit `, `_apply_normal_rowset_commit` and `_apply_compaction_commit`)? 
There are too many lines in one function.

Reply:
> How about refactor these three apply function (`_apply_column_partial_update_commit `, `_apply_normal_rowset_commit` and `_apply_compaction_commit`)? There are too many lines in one function.

I think it's unnecessary to complete the refactoring in this PR; it can be done in a new PR instead.